README.md

```diff
@@ -255,3 +255,7 @@ The plugin component should be added to the GraphiQL component in the `plugins`
 ### Design
 A design refactoring of the plugin is planned to improve the user experience. 
 The `figma` project with the definition can be found [here](https://www.figma.com/file/Qf8hgYpM5kUzItgVdesLbu)
+
+### E2E Testing
+
+- npx playwright test 
```

Comment:
- can we add this text using the code style: `npx playwright test`?
- is it worth adding more details on the E2E testing phase? Details about what it is testing or other useful information?

Reply:
Thanks, I have now done this